sdk/identity/azure-identity/src/main/java/com/azure/identity/AuthorizationCodeCredentialBuilder.java

```diff
@@ -49,21 +49,32 @@ public AuthorizationCodeCredentialBuilder redirectUrl(String redirectUrl) {
     }
 
     /**
-     * Sets whether to use an unprotected file specified by <code>cacheFileLocation()</code> instead of
-     * Gnome keyring on Linux. This is false by default.
+     * Allows to use an unprotected file specified by <code>cacheFileLocation()</code> instead of
+     * Gnome keyring on Linux. This is restricted by default.
      *
-     * @param allowUnencryptedCache whether to use an unprotected file for cache storage.
+     * @return An updated instance of this builder.
+     */
+    public AuthorizationCodeCredentialBuilder allowUnencryptedCache() {
```

Comment:
I don't believe we had added the support options for persisting the cache to `AuthorizationCodeCredential` in .NET and Python. @chlowell Was this in the design for the application authentication feature? If not perhaps we should add it to the backlog and remove from Java until we are ready to do this across languages.

Reply:
It was excluded because you need an authorization code to use this credential. If I have the record of a prior authentication, do I still need the authorization code? If I have an authorization code, why do I want to use cached data? Etc.

Reply:
I believe we've released a preview already with allowUnecnryptedCache on AuthorizationCredentialBuilder.
For consistency, we can remove it.
@jianghaolu  what are your thoughts ?

Reply:
This API is removed now.